tests/unit/features/partitions/test_lifecycle_manager.py

```diff
@@ -46,6 +46,33 @@ def valid_partitions_strategy():
     return strategy.filter(lambda partitions: "default" not in partitions[1:])
 
 
+def valid_namespaced_partitions_strategy():
```

Comment:
Does hypothesis allow generating an invalid list from the valid definition? It could be an interesting test too.

Reply:
@lengau, do you know?  I was wondering the same thing and it seemed like hypothesis is not designed for that.

Reply:
I don't think Hypothesis has a way to invert strategies. (I'm not sure how one could actually achieve this in the general case, though for text it might be more feasible.)